src/components/Feelings/Feelings.jsx

```diff
@@ -1,7 +1,7 @@
 import { FeelingsList } from "./FeelingsList";
 import { useState } from "react";
 import { Link } from "react-router-dom";
-import "./Feelings.css";
+import "./Feelings.module.css";

```

Comment:
You can't import CSS modules like that. You have to use : import classes from "./Feelings.module.css" - like that. And while we are using that particular class inside a className, you have to define it like classname={classes.main} - like that. Feel free to check out other jsx files where we have used CSS modules. Hope it will make sense.

Reply:
Ok I will see to it